File: Sem3/Advanced Methods of Programming/MyInterpreter - JavaFX - Final Project/project/src/domain/adt/MyPair.java
package domain.adt;


public class MyPair<T1, T2>
{
    private T1 first;
    private T2 second;


    public MyPair(T1 first, T2 second)
    {
        this.first = first;
        this.second = second;
    }


    public T1 getFirst()
    {
        return first;
    }


    public T2 getSecond()
    {
        return second;
    }


    @Override
    public String toString()
    {
        return first.toString() + " " + second.toString();
    }
}

File: Sem3/Advanced Methods of Programming/MyInterpreter - CSharp Console Application/MyInterpreter CSharp/domain/adt/MyPair.cs
namespace MyInterpreter_CSharp.domain.adt
{
    public class MyPair<T1, T2>
    {
        private T1 _first;
        private T2 _second;

        public MyPair(T1 first, T2 second)
        {
            _first = first;
            _second = second;
        }

        public T1 First => _first;

        public T2 Second => _second;

        public override string ToString()
        {
            return _first + " " + _second;
        }
    }
}